### DISCUSSION TITLE
Seems people do not working/maintain this project anymore, but why

### DISCUSSION THREAD

**Discussion by vincent-pli** [virtual-kubelet/virtual-kubelet#1009] [General]
I'm curious that, seems people do not working here anymore. I have a little confuse, actually I think this project is great and usful, is that the reason the project is marure? or good enough, may I missing something?

**Comment by cpuguy83** [comment]
@vincent-pli It's still being worked on.
If you are interested in some of the things currently being looked at: https://github.com/orgs/virtual-kubelet/projects/3
